stack.yaml

```diff
@@ -74,3 +75,6 @@ packages:
     - lib/vector-text
     - lib/yaml-utils
 
+extra-deps: [
```

Comment:
Extra deps don't live here. This should be added to the list in `config/snapshot.yaml` instead.